src/commands/init.js

```diff
@@ -80,6 +83,99 @@ async function handleDatabaseConfiguration() {
   return inquirer.prompt(promptContent);
 }
 
+function validateEndpoint(input) {
+  if (!VALIDATION_REGEX_URL.test(input)) {
+    return 'Application input must be a valid url.';
+  }
+  if (!VALIDATION_REGEX_HTTPS.test(input)) {
+    return 'HTTPS protocol is mandatory, except for localhost and 127.0.0.1.';
+  }
+  return true;
+}
+
+function getApplicationPortFromCompleteEndpoint(endpoint) {
+  if (endpoint.startsWith('http://localhost:') || endpoint.startsWith('http://127.0.0.1:')) {
+    return endpoint.substring(17);
+  }
+  return null;
+}
+
+async function getContentToAddInDotenvFile(config) {
+  const authSecret = await generateKey();
+  let contentToAddInDotenvFile = '';
+
+  if (config.applicationPort) {
+    contentToAddInDotenvFile += `APPLICATION_PORT=${config.applicationPort}\n`;
+  }
```

Comment:
[Misc] We don't handle the `APPLICATION_URL`?

Reply:
Well, I am not sure it is needed, as it not specified in the lumber generated projects.
Do you confirm?